charts/icm/values.yaml

```diff
@@ -1,7 +1,7 @@
 icm-as:
 
   image:
-    repository: intershophub/icm-as:11.0.5
+    repository: intershophub/icm-as:11.0.9
```

Comment:
The `ct install` test will fail here because 11.0.9 doesn't exist on intershophub at the moment. I think 11.0.8 already contains your changes and the test will pass through.